axis_backend/axis_backend/exception_handlers.py

```
@@ -0,0 +1,241 @@
+        'exception_message': str(exc),
+        'view': view.__class__.__name__ if view else 'Unknown',
+        'method': request.method if request else 'Unknown',
+        'path': request.path if request else 'Unknown',
+        'user': str(request.user) if request and hasattr(request, 'user') else 'Anonymous',
+        'status_code': response.status_code if response else 'Unknown',
+    }
+    
+    # Log at appropriate level
+    if response and response.status_code >= 500:
+        logger.error(
+            f"Server error: {log_context['exception_type']} in {log_context['view']}",
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)